### ISSUE TITLE
Message contents are lost when multitasking on iOS

### ISSUE THREAD

**Issue by binaryfire** [mattermost/mattermost-mobile#7485]
#### Summary
Often you’ll need to switch to another app in the middle of writing a message (eg. to copy something from a browser). Doing this on iOS results in the entire unfinished message being lost. This is very frustrating, especially when you’ve spent a few minutes composing something. No drafts are saved either.

### Environment Information
  - Device Name: iPhone X
  - OS Version: 16.5.1
  - Mattermost App Version: Latest
  - Mattermost Server Version: 8.01

#### Steps to reproduce
1. Write message (but don’t send). 
2. Flick up to bring up the iOS multi tasking app switcher.
3. Don’t close Mattermost. Just switch to a different app, or switch between a couple of apps.
4. Switch back to Mattermost. Message will be gone.

#### Expected behavior
Basically, this means multi tasking with Mattermost on iOS doesn’t work. None of the other apps I’ve used on iOS exhibit this behaviour and all allow picking up where you left off.
